content/docs/tasks/traffic-management/custom-gateway/index.md

```diff
@@ -0,0 +1,372 @@
+                readOnly: true
+              - mountPath: /etc/istio/ingressgateway-ca-certs
+                name: ingressgateway-ca-certs
+                readOnly: true
+              env:
+              - name: POD_NAME
+                valueFrom:
+                  fieldRef:
+                    apiVersion: v1
+                    fieldPath: metadata.name
+              - name: POD_NAMESPACE
+                valueFrom:
+                  fieldRef:
+                    apiVersion: v1
+                    fieldPath: metadata.namespace
+              - name: INSTANCE_IP
+                valueFrom:
+                  fieldRef:
+                    apiVersion: v1
+                    fieldPath: status.podIP
+              - name: ISTIO_META_POD_NAME
+                valueFrom:
+                  fieldRef:
+                    fieldPath: metadata.name
+          volumes:
+          - name: istio-certs
+            secret:
+              defaultMode: 420
+              optional: true
+              secretName: istio.istio-ingressgateway-service-account
+          - name: ingressgateway-custom-certs
+            secret:
+              defaultMode: 420
+              optional: true
+              secretName: istio-customingressgateway-certs
+          - name: ingressgateway-ca-certs
+            secret:
+              defaultMode: 420
+              optional: true
+              secretName: istio-ingressgateway-ca-certs
+          affinity:
+            nodeAffinity:
+              requiredDuringSchedulingIgnoredDuringExecution:
+                nodeSelectorTerms:
+                - matchExpressions:
+                  - key: beta.kubernetes.io/arch
+                    operator: In
+                    values:
+                    - amd64
+                    - ppc64le
+                    - s390x
+              preferredDuringSchedulingIgnoredDuringExecution:
+              - weight: 2
+                preference:
+                  matchExpressions:
+                  - key: beta.kubernetes.io/arch
+                    operator: In
+                    values:
+                    - amd64
+              - weight: 2
+                preference:
+                  matchExpressions:
+                  - key: beta.kubernetes.io/arch
+                    operator: In
+                    values:
+                    - ppc64le
+              - weight: 2
+                preference:
+                  matchExpressions:
+                  - key: beta.kubernetes.io/arch
+                    operator: In
+                    values:
+                    - s390x
+            podAntiAffinity:
+              requiredDuringSchedulingIgnoredDuringExecution:
+              - labelSelector:
+                  matchExpressions:
+                  - key: app
+                    operator: In
+                    values:
+                    - istio-internal-custom
+                topologyKey: kubernetes.io/hostname
+
+    {{< /text >}}
+
+1. Create your service:s
```

Comment:
Is this the right way to do this? As with deployment, can this be configured with helm?